### PR TITLE
Conflict with lp command

### PR DESCRIPTION
**Pull request by joshcangit**
There is also another `lp` command from CUPS in Linux.
I'm thinking of coming up with something other than `lp`.

### CODE REVIEW

**Comment by joshcangit** [comment]
I thought of choosing `lph` as the CLI.
If you have other suggestions, I might consider editing it.

**Comment by fisker** [comment]
That's what another bin for  https://github.com/fisker/left-phalange/blob/e5fc489d65ec068371949a17e17d4e132537562a/package.json#L29.
Please `left-phalange` instead.

**Comment by joshcangit** [comment]
> Please `left-phalange` instead.

I know, there's no problem with line 29 in the `**package.json**`.

I'm talking about the line 30 with `"lp"`.
That's the problem. It either has to be edited or removed.

**Comment by fisker** [comment]
I guess I understand your problem now, so you can't use `lp` to run "command from CUPS in Linux"?


If this is the problem, you'll have to find your own way to fix it.
If the problem is you can't use `lp` to run our tool, my answer is use `left-phalange` instead.

**Comment by fisker** [comment]
Closing, since we can't accept this anyway.

**Comment by joshcangit** [comment]
> I guess I understand your problem now, so you can't use `lp` to run "command from CUPS in Linux"?
> 
> 
> If this is the problem, you'll have to find your own way to fix it.
> If the problem is you can't use `lp` to run our tool, my answer is use `left-phalange` instead.

No.
I mean that because of this `lp` clashing with CUPS I cannot install `left-phalange` using `npm`.
This is the problem and not because I can't use `lp` so use `left-phalange`.
TLDR, your package is **uninstallable**.

**Comment by fisker** [comment]
Alright, thanks for your explanation. But I don't know how it npm works on your OS, maybe you can try install npm packages to a different location? Maybe `npm config set prefix`? There has to be a solution, but I don't know.
